handlers/handlers.go

```diff
@@ -63,7 +63,7 @@ func init() {
 func loginURL(r *http.Request, state string) string {
 	// State can be some kind of random generated hash string.
 	// See relevant RFC: http://tools.ietf.org/html/rfc6749#section-10.12
-	var url = ""
+	var theUrl = ""
```

Comment:
why "theUrl"?

Reply:
My IDE  was telling me those were names overlapping with some builtin stuff. So it was the simplest way to change them.  They can be anything but what they were.